zebrad/tests/acceptance.rs

```diff
@@ -445,21 +445,6 @@ fn ephemeral(cache_dir_config: EphemeralConfig, cache_dir_check: EphemeralCheck)
     Ok(())
 }
 
-#[test]
-fn app_no_args() -> Result<()> {
```

Comment:
New features need a test to make sure they keep working.

The test can be similar to `start_no_args()`, but it might need to be called from `config_test()` to avoid port conflicts.

Reply:
Calling it from `config_test()` to avoid port conflicts makes sense to me, I think the `start_no_args()` and `start_args()` should be run from there as well?

Reply:
> `zebra_zcash_listener_conflict()` still failed so I added `start_no_args()` and `start_args()` there too.

I'm not sure why those failures are happening for these tests, because:

They are configured to run on separate ports:
https://github.com/ZcashFoundation/zebra/blob/87f4308caf0734a0cb163bba40107b85026a0ed1/zebrad/tests/common/config.rs#L31-L32

Running tests on different ports in parallel makes sure that the config is actually being applied. If the config isn't applied, all the Zebra instances run on the default ports, causing a port conflict and test failure.

This is passing on Linux and macOS in CI. So I'm wondering if it is platform-specific.
(Zebra doesn't support Windows or WSL and the moment, and we have minimal support for macOS.)

Can you show me the info-level logs from each test failure?

You might need to do:
```sh
export RUST_LOG=info
```
Before running the tests.

Reply:
After re-running the tests with the same code, this was almost certainly a different mistake on my end. `zebra_zcash_listener_conflict()` succeeds consistently and the info logs are what they normally are when we run zebrad.